drivers/SmartThings/zigbee-fan/profiles/fan-light.yml

```diff
@@ -0,0 +1,27 @@
+name: fan-light
+components:
+  - id: main
+    label: Fan 
+    capabilities:
+      - id: switch
+        version: 1
+      - id: fanSpeed
+        version: 1
```

Comment:
if you want to restrict this to 3 speeds, you could set the range like this:

```
    config:
      values:
        - key: "fanSpeed.value"
          range: [0, 3]
```

We also use this for most zigbee dimmers as well: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/drivers/SmartThings/zigbee-switch/profiles/on-off-level.yml#L10

and I would suggest you also use it in the light component below.

Reply:
I will change it.